Form-Controls/index.html

```diff
@@ -7,21 +7,73 @@
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <section>
+      <label for="name">Name</label>
+      <input id="name" type="text" minlength="3" maxlength="8" required />
+    </section>
+    <section>
+    <label for="email">Email</label>
+    <input id="email" type="email" required/>
+    </section>
+    <section>
```

Comment:
Oh look, here it is!

Do you need both section and fieldset here? What is section doing for you that fieldset is not?